kura/test/org.eclipse.kura.nm.test/src/test/java/org/eclipse/kura/nm/NMModemResetHandlerTest.java

```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.nm;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import org.freedesktop.dbus.exceptions.DBusException;
+import org.freedesktop.modemmanager1.Modem;
+import org.freedesktop.networkmanager.Device;
+import org.junit.Test;
+
+public class NMModemResetHandlerTest {
+
+    private Modem mockMMModemDevice = mock(Modem.class);
+    private NMModemResetHandler resetHandler = null;
+
+    private static final String MOCK_DEVICE_DBUSPATH_9 = "/org/freedesktop/NetworkManager/Devices/9";
+    private static final String MOCK_DEVICE_DBUSPATH_5 = "/org/freedesktop/NetworkManager/Devices/5";
+
+    @Test
+    public void modemShouldNotBeResetWhenModemDoesntDisconnect() {
+        givenNMModemResetHandlerWith(MOCK_DEVICE_DBUSPATH_9, this.mockMMModemDevice, 50);
+
+        whenThreadHasWaitedFor(1000);
+
+        thenModemWasReset(false);
+    }
+
+    @Test
+    public void modemShouldNotBeResetWhenADifferenctModemDisconnects() throws DBusException {
```

Comment:
typo

Reply:
Fixed in: d16bb6e238e057a6c26346aaa61849e62d496f2a